docs/api.rst

```diff
@@ -167,6 +167,16 @@ data, represented as xugrid objects.
        OverlapRegridder
        RelativeOverlapRegridder
 
+Gridding
+--------
+
+Gridding is the process of converting 1d networks to 2d grids. 
```

Comment:
I dislike the term "1d network". I think better is something like: "linear 2D networks (UGRID1D)" or something. And maybe "2d grids" should be "2D grids (structured, or UGRID2d)".

Reply:
The UGRID convention refers to "1D network" https://ugrid-conventions.github.io/ugrid-conventions/#1d-network-topology

Reply:
Fair point, if people are familiar with the UGRID conventions they'll know, so this is generally fine (e.g. changelog). Still I think a less terse description would be good here since people will be searching the API reference for functions and methods.
So maybe write explicitly "a network of line elements" or something?

Reply:
I see, changed to "a network of line elements"